<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="FidelBanksApp2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>591</width>
      <height>361</height>
     </rect>
    </property>
    <widget class="QWidget" name="Main">
     <widget class="QLabel" name="lblAdmin">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Admin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAdminName">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAdminPassword">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblAdminName">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAdminPassword">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdmin">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>150</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblUser">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>190</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>User</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbluserpassword">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Pasword</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblUserName">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leUserName">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>230</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leUserpassword">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUser">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>290</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Admin">
     <widget class="QPushButton" name="btnAdminWithdraw">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Withdraw</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdminDeposit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Deposit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdminchUserdata">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>chUserData</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdminTransfer">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Transfer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdminAddUser">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>AddUser</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdminDeleteUser">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>DeleteUser</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdminlogout">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>290</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAdNameadminpg">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Admin Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAdNameAdminpg">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="adminpgsearchbtn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>search Account Number</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="User">
     <widget class="QPushButton" name="btnUserWithdraw">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Withdraw</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUserDeposit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Deposit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnchUseruserdata">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>chUserdata</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUserTransfer">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Transfer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUserLogout">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAccNumusrpg">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Account Number</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrenBalusrpg">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Balance</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbltranshistoryusrpg">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Transaction History</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAcusrpg">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Acc Num</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAmountusrpg">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>240</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leuserNameusrpg">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAccnumusrpg">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leCurrentBalusrpg">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAc1usrpg">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>260</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leamount1usrpg">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>260</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAc2usrpg">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>290</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAmount2usrpg">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>290</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAc3usrpg">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>320</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAmount3usrpg">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>320</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbluserNameusrpg">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="UserData">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Change User Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>User Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Phone Number</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>180</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Withdraw">
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{font-size:30px;}</string>
      </property>
      <property name="text">
       <string>Withdraw</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{font-size:15px;
color:blue;}</string>
      </property>
      <property name="text">
       <string>Amount : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_14">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="okWithdrawBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>210</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{font-size:15px;color:blue;}</string>
      </property>
      <property name="text">
       <string>ok</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExitWithdraw">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>210</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{font-size:15px;color:blue;}</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Transfer">
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Transfer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Account to Transfer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>200</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Balance</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AddNewUser">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD USER</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblusernameadduserpg">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbluserpasswordadduserpg">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>UserPassword</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblphonenumadduserpg">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Phone Number</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbladdressadduserpg">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>230</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leuserNameadduserpg">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leuserpasswordadduserpg">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lephonenumadduserpg">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>190</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leaddressadduserpg">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DeleteUser">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>DELETE USER</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Account Number</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Deposit">
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{font-size:30px;}</string>
      </property>
      <property name="text">
       <string>Deposit</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAmountdepopg">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{font-size:15px;color:blue;}</string>
      </property>
      <property name="text">
       <string>Amoun to Deposit:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAmountDepopg">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="depositFundBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>220</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>deposit fund</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>220</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{font-size:30px;color:blue;}</string>
    </property>
    <property name="text">
     <string>Banking System</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
